api/v1alpha/example_client.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Copyright 2016 Author Josef Boudnik - josef.boudnik@seznam.cz
```

Comment:
As per https://github.com/coreos/rkt/pull/2333#discussion_r57760642 would you mind to adjust the Copyright?